README.md

```diff
@@ -1,35 +1,43 @@
----
```

Comment:
good call to remove this header so that the homepage isn't showing up on Code Samples site